packaging/win/delugewin.spec

```diff
@@ -149,7 +149,7 @@ for e, d in executables.items():
         hookspath=[],
         hooksconfig={},
         runtime_hooks=runtime_hooks,
-        excludes=[],
+        excludes=['gi.repository.Gst', 'gi.repository.GstAudio', 'gi.repository.GstVideo', 'gi.repository.GstBase', 'gi.repository.GstPbutils'],
```

Comment:
I don't see `gstreamer` as part of gvsbuild, why do we need this? If it is included then likely better to not build it in the first place by adding `gstreamer` to the [skip step](
https://github.com/deluge-torrent/gvsbuild-release/blob/5c3100a8726af2945915029673853b70aa616373/.github/workflows/release.yml#L50)

Reply:
Gstreamer project isn't built, but for some reason some references to it is picked up by pyinstaller through gi.repository. This was same in pyinstaller 4.x and displayed a "traceback" error in cmd-output during freeze, but the freezing continued till end succesfully regardless - this was one of the things they changed/fixed in pyinstaller 5.x i.e. implememnted a new child/parent method which would now correctly block/quiet the run(main-loop I gather) whenever some sub-process failed.

This fix/workaround is fixing a "mssing typelib" error for various gst components for both pyinstaller 4.x and 5.x, but just now doesn't quit the process in error litle after start.

I have been trying to see if could find the culprit specifically so could report to gvsbuild devs, but haven't yet though.

When you asked if I could clarify, then did you refer to the commit-message, or to you(like here)? Sorry for being dense :)

I understand of-course if you close this, but was also just to note(as you have no "issues" section here) that either need something like this, or, need stipulate 'pip install pyinstaller==4.10' to produce a freezed build currently.

Thanks fo taking a look :)

Reply:
It is always good to include the reason why something has changed in the commit message, that way in the future we can use `git blame` to see why the change was made.

As for the error, I don't see any issues with the CI builds, is this a local build issue? What are your steps to reproduce? 

![image](https://user-images.githubusercontent.com/606038/173234632-2f4e8383-f212-442f-ac78-17245acb5331.png)

I am happy to merge the Readme changes but without more details on the error I am reluctant to include that change